src/fmu/dataio/_model/global_configuration.py

```diff
@@ -92,16 +92,28 @@ class StratigraphyElement(BaseModel):
     """
 
     name: str
-    stratigraphic: bool
-    alias: Optional[List[str]] = Field(default=None)
+    stratigraphic: bool = Field(default=False)
+    alias: Optional[List[str]] = Field(default_factory=list)
     stratigraphic_alias: Optional[List[str]] = Field(default=None)
+    offset: float = Field(default=0.0, allow_inf_nan=False)
+    top: Optional[data.Layer] = Field(default=None)
+    base: Optional[data.Layer] = Field(default=None)
 
     @field_validator("alias", "stratigraphic_alias", mode="before")
     @classmethod
     def _prune_nones(cls, values: Any) -> Any:
         # For backwards compatibility.
         return None if values is None else [v for v in values if v is not None]
 
+    @field_validator("top", "base", mode="before")
+    @classmethod
+    def _set_name_attribute_if_string_input(
+        cls, value: Dict | str | None
+    ) -> Dict | None:
+        if isinstance(value, str):
+            return {"name": value}
```

Comment:
Does it make sense to return `Layer | None` and do `Layer(name=value)` or `Layer(**value)` if dict here instead? Or maybe it is fine to let Pydantic do this anyway? The end result is the same, I am not sure whether we should prefer one over the other. What do you think?

Reply:
I did not think of that.. 🙂  
but maybe `Layer.model_validate(value)` instead of  `Layer(**value)` to get proper validation error out to the user in case they provide an attribute not present in `Layer`. 

Thinking about it I do tend to prefer the validation error coming from the top level (i.e. validating the entire config), it is a bit more specific in pointing out where in the nested structure your error occur.. if we do the validation on a specific Layer here, it will not be indacative of where in the config e.g. what list index it is etc.

So I'll stick with returning the `dict` here 👍